include/cantera/oneD/StFlow.h

```diff
@@ -325,22 +322,112 @@ class StFlow : public Domain1D
+    * on \f$ \dot{m} \f$ from the inlet. The default value for the continuity equation
+    * is zero velocity (\f$ u \f$) at the left and right boundary.
+    */
+    virtual void evalContinuity(double* x, double* rsd, int* diag,
+                                double rdt, size_t jmin, size_t jmax);
+
+    /**
+    * @brief Evaluate the momentum equation residual.
+    *
+    * The function calculates the radial momentum equation defined as
+    * \f[
+    *    \rho \frac{dV}{dt} + \rho u \frac{dV}{dz} + \rho V^2 =
+    *    \frac{d(\mu \frac{dV}{dz})}{dz} - \lambda
+    * \f]
+    *
+    * @details The radial momentum equation incorporates terms for temporal and spatial
+    * variations of radial velocity (\f$ V \f$). The default boundary condition is zero
+    * radial velocity (\f$ V \f$) at the left and right boundary.
+    */
+    virtual void evalMomentum(double* x, double* rsd, int* diag,
                               double rdt, size_t jmin, size_t jmax);
 
+    /**
+    * @brief Evaluate the energy equation residual.
+    *
+    * The function calculates the energy equation:
+    * \f[
+    *    \rho c_p \frac{dT}{dt} + \rho c_p u \frac{dT}{dz} =
+    *    \frac{d(k \frac{dT}{dz})}{dz} - \sum_k (\omega_k h_{k_{\text{ref}}}) -
+    *    \sum_k \left( \frac{J_k c_{p_k}}{M_k} \frac{dT}{dz} \right)
+    * \f]
+    *
+    * @details The energy equation includes terms for temporal and spatial
+    * variations of temperature (\f$ T \f$). It includes contributions from
+    * chemical reactions and diffusion. Default is zero temperature (\f$ T \f$)
+    * at the left and right.
+    */
+    virtual void evalEnergy(double* x, double* rsd, int* diag,
+                            double rdt, size_t jmin, size_t jmax);
+
+    /**
+    * @brief Evaluate the species equations' residuals.
+    *
+    * The function calculates the species equations as
+    * \f[
+    *    \rho \frac{dY_k}{dt} + \rho u \frac{dY_k}{dz} + \frac{dJ_k}{dz} = M_k\omega_k
+    * \f]
+    *
+    * @details The species equations include terms for temporal and spatial variations
+    * of species mass fractions (\f$ Y_k \f$). The default boundary condition is zero
+    * flux for species at the left and right boundary.
+    */
+    virtual void evalSpecies(double* x, double* rsd, int* diag,
+                             double rdt, size_t jmin, size_t jmax);
+
+    /**
+    * @brief Evaluate the lambda equation residual.
+    *
+    * The function calculates the lambda equation as
+    * \f[
+    *    \frac{d\lambda}{dz} = 0
+    * \f]
+    *
+    * @details The lambda equation serves as an eigenvalue that allows the momentum
+    * equation and continuity equations to be simultaneously satisfied. The lambda
+    * equation propgates information from left-to-right. The default
+    * boundary condition is zero (\f$ \lambda \f$) at the left and zero flux at
+    * the right boundary.
+    */
+    virtual void evalLambda(double* x, double* rsd, int* diag,
```

Comment:
move up (below `evalMomentum`), so the order is consistent with the state vector.